web/client/themes/default/less/timeline.less

```diff
@@ -196,6 +196,9 @@
         }
     
         .vis-timeline {
+            .vis-item.interval {
+                height: 10px;
+            }
```

Comment:
these changes should not be included in the 
// **************
// Theme
// **************
part

these are more layout changes
please move them in the 
// **************
// Layout section
// **************

Reply:
moved to theme section